aws_s3_rules/aws_s3_unauthenticated_access.py

```diff
@@ -3,7 +3,7 @@
 
 
 def rule(event):
-    return event.get('bucket') in AUTH_BUCKETS and 'requester' not in event
+    return event.get('bucket') in AUTH_BUCKETS and not event.get('requester')
```

Comment:
Usage of `in` was having interesting side effects 🤔 

I'm not quite sure why, but this fix solved it